### DISCUSSION TITLE
Can't deploy on Vercel using latest Next.js canary

### DISCUSSION THREAD

**Discussion by kelihansen** [vercel/community#4499] [Help] [answered]
### Summary

My Vercel builds are failing with `error next@13.5.6-canary.1: The engine "node" is incompatible with this module. Expected version ">=18.18.2". Got "18.17.1"`. If there's a way for me to specify that the very latest Node version should be used for my deployments, I'm not finding it 🧐

### Steps to Reproduce

I would think y'all had done this, so I'm hopeful maybe I'm just being clueless, but try to deploy an app with `next@13.5.6-canary.1` on Vercel?

**Comment by swarnava** [comment, marked as answer]
Hi,

We don't support 18.18.2 yet, the current supported version is 18.17.1 which we will upgrade eventually. You should always use the major version incase you want to pin node version. i.e 18.x: https://vercel.com/docs/functions/serverless-functions/runtimes/node-js

**Comment by kelihansen** [comment]
Then is there any chance [this](https://github.com/vercel/next.js/pull/56896) can be changed to a version Vercel supports?

**Comment by kelihansen** [comment]
In case anyone else finds this, I got around the issue for now by adding a `.yarnrc` file containing only the line `--install.ignore-engines true` to the root of my project
